sos/plugins/networking.py

```diff
@@ -137,8 +137,7 @@ def setup(self):
         self.add_forbidden_path("/proc/net/cdp")
         self.add_forbidden_path("/sys/net/cdp")
         # Dialogic Diva
-        self.add_forbidden_path("/proc/net/eicon/di*")
-        self.add_forbidden_path("/proc/net/eicon/adapter*/dynamic_l1_down")
+        self.add_forbidden_path("/proc/net/eicon/*")
```

Comment:
I'd rather just drop the entire tree ("/proc/net/eicon"); although rare, there have been drivers that created ".dotfiles" in proc paths in the past.